### PR TITLE
feat(wallet): Add ToggleSwitch to transfer pages & new images from design added & b…

### LINKED ISSUE #118: Update Transfer pages
Add ToggleSwitch to pay/send pages.
Add design/image elements to pay/send/request pages.
Update Bubbles image.
Fis smaller outline issues, menu link

### PR DESCRIPTION
**Pull request by Tymmmy**
### Context
- fixes #118

### Changes

- added ToggleSwitch to send/pay pages
- added new images from designs, updated some old ones
- added Interledger title to layout
- fixed some outline bugs, Menu link bug
- added some extra info to contributing 
- added Button hover styles for primary

### CODE REVIEW

File: packages/wallet/src/pages/index.tsx [hunk 1]
```diff
@@ -65,6 +66,7 @@ export default function Home({ accounts }: HomeProps) {
           ))}
         </div>
       </div>
+      <SmallBubbles className="absolute inset-x-0 bottom-0 block w-full lg:hidden" />
```
Comment (raducristianpopa) [inline packages/wallet/src/pages/index.tsx:69]: This is going over the account cards.
<details>
<summary>Preview </summary>
image

</details>

File: packages/wallet/src/ui/Button.tsx [hunk 2]
```diff
@@ -25,7 +25,9 @@ import { ButtonOrLink, type ButtonOrLinkProps } from './ButtonOrLink'
 const buttonStyles = cva(['inline-flex items-center justify-center'], {
   variants: {
     intent: {
-      primary: ['text-white bg-gradient-to-r from-green-5 to-green-6'],
+      primary: [
+        'text-white bg-gradient-to-r from-green-5 to-green-6 hover:from-[#7acebe] hover:to-[#2b7576] hover:shadow-md'
```
Comment (raducristianpopa) [inline packages/wallet/src/ui/Button.tsx:29]: Should we add these colors to tailwind config?🤔

File: packages/wallet/src/ui/Button.tsx [hunk 3]
```diff
@@ -25,7 +25,9 @@ import { ButtonOrLink, type ButtonOrLinkProps } from './ButtonOrLink'
...
+      ],
       secondary: ['text-white bg-gradient-secondary'],
       success: ['text-green-3 bg-white'],
       error: ['text-pink bg-white'],
```
Comment (raducristianpopa) [inline packages/wallet/src/ui/Button.tsx:33]: Let's add the hover state to all the buttons.

File: packages/wallet/src/ui/Button.tsx (hunk 2 above)
Comment (Tymmmy) [reply packages/wallet/src/ui/Button.tsx:29]: I will add them

**Review by raducristianpopa — APPROVED** [review]

**Pull request merged**